django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
Ouch. I think that's not what we should do. I'm running tests with the following tests right now:
```diff
diff --git a/django/db/models/fields/__init__.py b/django/db/models/fields/__init__.py
index de13b73..b91c604 100644
--- a/django/db/models/fields/__init__.py
+++ b/django/db/models/fields/__init__.py
@@ -1268,7 +1268,7 @@ class DateField(DateTimeCheckMixin, Field):

     def pre_save(self, model_instance, add):
         if self.auto_now or (self.auto_now_add and add):
-            value = datetime.date.today()
+            value = timezone.now().date()
             setattr(model_instance, self.attname, value)
             return value
         else:
@@ -2268,7 +2268,7 @@ class TimeField(DateTimeCheckMixin, Field):

     def pre_save(self, model_instance, add):
         if self.auto_now or (self.auto_now_add and add):
-            value = datetime.datetime.now().time()
+            value = timezone.now().time()
             setattr(model_instance, self.attname, value)
             return value
         else:
```